headertest/dummy_header.go

```diff
@@ -19,7 +19,7 @@ var ErrDummyVerify = errors.New("dummy verify error")
 type DummyHeader struct {
 	Chainid      string
 	PreviousHash header.Hash
-	HeightI      int64
+	HeightI      uint64
```

Comment:
random but why `HeightI` ? and not `Height`

Reply:
Becuase you can't have fields and methods of the same name